pkg/vpcmodel/explainabilityInput.go

```diff
@@ -28,6 +27,101 @@ func (e *ExplanationArgs) Dst() string {
 	return e.dst
 }
 
+const (
+	noErr                        = iota
+	exitNowErr                   // exit now with the error (do not wait until we go over all vpcs)
```

Comment:
The objective of this const is not to described the error message, but to say describe how to treat an error detected in a single vpc context. exitNowErr captures errors that implies immediate exit, exactly as said.
I'll add documentation to make this clearer

Reply:
give the error a name that describe the error, and not a name the action that need to be done.

in general, try to give names that describe the element that you name.
for example, "VpcsConfigsMap" describe a multi configuration, the fact that it is a map is not relevant.
same goes here. the name should describe the error, not the action. if you can not find a specific name, you can just call it fatal, severe or what ever describe it.
the action that is done later, is not relevant for the name. the action might be changed later. (do we exit? do we move to the next vpc? in case of multi explanation, do we move to the next explanation?)

Reply:
Thanks for the explantation.
I'm renamed it to fatalErr and updated the documenation